naresh/src/main/java/com/lftechnology/phpjava/ems/utlis/Router.java

```diff
@@ -0,0 +1,112 @@
+package com.lftechnology.phpjava.ems.utlis;
+
+import com.lftechnology.phpjava.ems.constants.Constant;
+import com.lftechnology.phpjava.ems.controllers.EmployeeController;
+import com.lftechnology.phpjava.ems.controllers.LoginController;
+import com.lftechnology.phpjava.ems.services.EmployeeService;
+import com.lftechnology.phpjava.ems.views.CommonViewUtility;
+
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * RouterService
+ *
+ * @author Naresh Maharjan <nareshmaharjan@lftechnology.com>
+ * @since August, 09 2016
+ */
+public class Router {
+
+    private static LoginController loginController = new LoginController();
+    private static EmployeeController employeeController = new EmployeeController();
+    private static CommonUtility commonUtility = new CommonUtility();
+
+    static {
+        try {
+            onRouteBootstrap();
+        } catch (SQLException e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS1166)